README.md

```diff
@@ -63,17 +64,41 @@ workflow.display()
 
 ```
 
+### Make a FairStep from scratch
+```python
+from fairworkflows import FairStep, FairWorkflow, fairstep
+import rdflib
 
-### Example
-* See [examples/test_fairworkflows.ipynb](examples/test_fairworkflows.ipynb) for a current example of using the fairworkflows library to build a workflow using plex rdf
+# Make a new 'empty' step
+step = FairStep()
+
+# Specify various characteristics needed to describe it
+step.label = 'Label for this step'
+step.description = 'String describing the procedure that happens in this step'
+step.is_manual_task = True
+
+# Add other statements, about the step itself
+step.set_attribute(rdflib.URIRef('www.example.org/pred'), object=rdflib.URIRef('www.example.org/obj'))
+
+# Add any other, general triples
+step.add_triple(rdflib.URIRef('www.example.org/subj'), rdflib.URIRef('www.example.org/pred'), rdflib.URIRef('www.example.org/obj'))
 
-### Notes
-The np script needs to be called to generate rsa keys in the `~/.nanopub` directory.
+# Set the URIs of the inputs and outputs to this step
+step.inputs = ['www.example.org/input1', 'www.example.org/input2']
+step.outputs = ['www.example.org/output2']
```

Comment:
I would actually like a more real-world but still toy-example. Slicing an union or something like that.

Reply:
I did exactly that: Slicing an union. It is an actually working example, see: [http://test-server.nanopubs.lod.labs.vu.nl/RA6ieQhLZMhIj-zS9TyLPHCqGbCVTGkwr--6Yhp28394I](the resulting nanopub)